DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -9920,8 +9922,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/DuckDuckGo/BrowserServicesKit";
 			requirement = {
-				kind = exactVersion;
-				version = 112.0.0;
+				branch = "daniel/subcription.user-cancel";
```

Comment:
This will be updated once the BSK PR is merged and BSK is released

Reply:
I may have missed something here, this PR is open and is against Main, are you going to merge it now (soon) or wait for BSK PR? if you need to wait for BSK then you should tag the PR and follow the process in https://app.asana.com/0/1202500774821704/1200837094583426/f

Reply:
@federicocappelli This is what we normally do for BSK changes

1.  We open 3 PRs.  BSK, macOS and iOS.  
2. Both macOS point to the BSK branch for testing
3. Once everything approved, we release BSK, and update the macOS and iOS PRs to point to that release
4. We merge both iOS and MacOS PR's together.

This PR is referenced in the BSK PR I mentioned.

Reply:
Ok, I understand where the problem is here, I'm not used to the fact that a PR can be changed after is approved, and you want me to approve this and then change the BSK reference, I'm ok with that, I'm just not sure why the BSK reference is not already in when the BSK PR has been approved.
LGTM anyway

Reply:
@federicocappelli  As mentioned in MM, we often approve before releasing BSK, so the author can quickly merge all three PR's async. (macOS and iOS PRs are often opened by different people)

I've just pointed to 113.1.0 release in case you want to have a look before merging

Reply:
I'm good with that, free to merge